api_core/google/api_core/grpc_helpers.py

```diff
@@ -161,7 +212,7 @@ def create_channel(target, credentials=None, scopes=None, **kwargs):
             service. These are only used when credentials are not specified and
             are passed to :func:`google.auth.default`.
         kwargs: Additional key-word args passed to
-            :func:`google.auth.transport.grpc.secure_authorized_channel`.
+            :func:`_create_secure_channel`.
```

Comment:
since this is a public method, we shouldn't reference non-public methods. Maybe change to this `grpc.secure_channel`?

Reply:
Ack.